src/App.js

```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import calculator from './components/calculator';
 
 function App() {
   return (
```

Comment:
Same here, The `App.js` component should also be class-based. Kindly refactor your component to use classes as per project requirements.